test/extended/cluster/cl.go

```diff
@@ -30,7 +30,7 @@ var rootDir string
 var _ = g.Describe("[Feature:Performance][Serial][Slow] Load cluster", func() {
 	defer g.GinkgoRecover()
 	var (
-		oc                = exutil.NewCLI("cl", exutil.KubeConfigPath())
+		oc                = exutil.NewCLIWithoutNamespace("cl")
```

Comment:
There is some potential breakage using `NewCLIWithoutNamespace()` as it doesn't set `c.configPath`. This is why for some functions we'll have to run `oc.AsAdmin()`; to set it manually.